user_manual/conf.py

```diff
@@ -43,13 +43,13 @@
 master_doc = 'index'
 
 # The short X.Y version.
-version = '1.9.1'
+version = '2.0'
 # The full version, including alpha/beta/rc tags.
-release = '1.9.1'
+release = '2.0'
```

Comment:
For consistency throughout the project format of version should always be three-fold x.y.z, so 2.0.0 here.